ISA-XLSX.md

```diff
@@ -164,19 +133,19 @@ This section MUST contain the following labels, with the specified datatypes for
-| Investigation Publication Status Term Accession Number | String or URI                                                                                      | The accession number from the Term Source associated with the selected term.                                                                                                               |
+| Investigation Publication PubMed ID                                | String formatted as valid PubMed ID                                                                | The PubMed IDs of the described publication(s) associated with this investigation.                                                                              |
+| Investigation Publication DOI                          | String formatted as valid DOI                                                                      | A Digital Object Identifier (DOI) for that publication (where available).                                                                                 |
+| Investigation Publication Author List                  | String                                                                                      | The list of authors associated with that publication.                                                                                |
+| Investigation Publication Title                        | String                                                                                             | The title of publication associated with the investigation.                                                                              |
+| Investigation Publication Status                       | String, or Ontology Annotation by providing accompanying Term Accession Number and Term Source REF | A term describing the status of that publication (i.e. submitted, in preparation, published).                                                                                 |
+| Investigation Publication Status Term Accession Number | String or URI                                                                                      | The accession number from the Term Source associated with the selected term.                                                                                       |
 | Investigation Publication Status Term Source REF       | String                                                                                             | Identifies the controlled vocabulary or ontology that this term comes from. The Source REF has to match one the Term Source Name declared in the in the Ontology Source Reference section. |
 
-For example, the `INVESTIGATION PUBLICATIONS` section of an ISA-Tab `i_*.txt` file may look as follows:
+For example, the `INVESTIGATION PUBLICATIONS` section of an ISA-XLSX `isa.investigation.xlsx` file may look as follows:
 
 ```default
 INVESTIGATION PUBLICATIONS
```

Comment:
maybe do these blocks as table. As different columns are very difficult to recognize in a code block.

Reply:
done, although i really do not like the examples from the initial ISA specs. the descriptions are far too large. I get where they come from, their sample should represent a 'real' investigation description. but that huge text block does really not help. I think @HLWeil will adapt the samples anyway.